src/main/java/seedu/jarvis/logic/parser/planner/AddTaskCommandParser.java

```diff
@@ -53,11 +53,15 @@ public AddTaskCommand parse(String args) throws ParseException {
         String taskType = argMultimap.getValue(PREFIX_TASK_TYPE).get();
         String taskDes = argMultimap.getValue(PREFIX_TASK_DES).get();
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
-        System.out.println("hello there");
         LocalDate[] dates = new LocalDate[2];
         if (isEventOrDeadline(argMultimap)) {
-            System.out.println("hi there");
```

Comment:
Lets avoid printing calls for testing methods. you may forget to delete these calls and would leave very weird information being printed out in production code.

The printing can be useful, but you can use assertions instead as breakpoints.